DOC-2610-Added an endpoint for fetching metadata from snowflake data source; [4.2.0-dev] (#263)
tigergraph/gsql-docs: modules/API/pages/gsql-endpoints.adoc

--
====

'''

> === get metadata from a Snowflake data source

Reviewer: ```suggestion
=== get schema information from a Snowflake data source

```

`GET /gsql/v1/metadata/data-sources/{dsName}`

> This endpoint is used to get metadata from a Snowflake data source.

Reviewer: ```suggestion
This endpoint is used to get schema information from a Snowflake data source.
`{dsPath}` is the name of a previously created Snowflake data source object.

```

> ==== Parameters:

Reviewer: ```suggestion
==== Query Parameters:

```
